### ISSUE TITLE
Improve edit pane visibility.

### ISSUE THREAD

**Issue by bouncyball-git** [ilia3101/MLV-App#118]
Could it be better if make the titles of edit panel a bit different color. Yellow maybe?

**Comment by masc4ii** [comment]
![bildschirmfoto 2018-10-23 um 10 13 45](https://user-images.githubusercontent.com/30245296/47345974-5d219100-d6ac-11e8-937e-62bce4c24928.png)
?

**Comment by bouncyball-git** [comment]
It really improves visibility isn't it?

**Comment by masc4ii** [comment]
I don't know... it is... so colorful?! 😄

**Comment by bouncyball-git** [comment]
Can you make commit? I'll check :)

**Comment by masc4ii** [comment]
Go to darkstyleOSX.qss (if you are on OSX, else darkstyle.qss) and search for "QGroupBox::title". There you insert this "color:..." line :
```
QGroupBox::title{
  color: rgb(255, 249, 162);
...
```

**Comment by bouncyball-git** [comment]
This does not change anything ...

**Comment by masc4ii** [comment]
Did you compile again?

**Comment by bouncyball-git** [comment]
Yeah cleaned all and compiled :(

**Comment by masc4ii** [comment]
Are you maybe at `QGroupBox{` and not `QGroupBox::title{` ? It is line 137, where to insert the color. If you are on Linux, in darkstyle.qss (without the OSX in filename).

**Comment by bouncyball-git** [comment]
other files/dark style/darkstyle.qss, line 160 (not 137):

```
QGroupBox::title{
  color: rgb(255, 249, 162);
  subcontrol-origin:margin;
  subcontrol-position:left top;
  padding:4px 6px;
  margin-left:3px;
  background-color:qlineargradient(x1:0,y1:1,x2:0,y2:0,stop:0 rgba(25,25,25,127),stop:1 rgba(53,53,53,75));
  border:1px solid rgba(25,25,25,75);
  border-top-left-radius:4px;
  border-top-right-radius:4px;
}
```

**Comment by masc4ii** [comment]
Yes... looks identical to what I have here.

**Comment by bouncyball-git** [comment]
Deleted build directory entirely. Compiled again. Does not work :(

**Comment by bouncyball-git** [comment]
Why do we have the line difference?

**Comment by bouncyball-git** [comment]
I have another idea. What if make group triangle be different color not title itself? Will be less colorful and we can try different colors.

**Comment by masc4ii** [comment]
This difference is because of Groupbox title margins - Windows interprets it different - whyever. I couldn't solve it easier.
Should the triangle maybe be closer to white?

**Comment by masc4ii** [comment]
😄 hmmm... so... more Lightroom feeling?
![bildschirmfoto 2018-10-23 um 19 40 01](https://user-images.githubusercontent.com/30245296/47379550-80276180-d6fb-11e8-87bd-5ee7abfd05c9.png)

**Comment by bouncyball-git** [comment]
Well it looks nice can I test this here?

**Comment by masc4ii** [comment]
![bildschirmfoto 2018-10-24 um 10 57 19](https://user-images.githubusercontent.com/30245296/47419036-a42a8780-d77b-11e8-85a6-83840eb4357c.png)
In MainWindow.ui, search for scrollArea, search for StyleSheet, and add these lines there.

**Comment by masc4ii** [comment]
![bildschirmfoto 2018-10-24 um 11 18 02](https://user-images.githubusercontent.com/30245296/47420377-90cceb80-d77e-11e8-9c4d-13c1be4d3f93.png)
😟

**Comment by masc4ii** [comment]
![bildschirmfoto 2018-10-24 um 11 26 50](https://user-images.githubusercontent.com/30245296/47420885-c1f9eb80-d77f-11e8-9c12-6faee066df3b.png)
😟 
```
QGroupBox::title{
  ...
  background-color:qlineargradient(x1:0,y1:1,x2:0,y2:0,stop:0 rgba(24,76,128,255),stop:1 rgba(42,130,218,255));
}
```
Looks like made by a clown too...

**Comment by masc4ii** [comment]
![bildschirmfoto 2018-10-24 um 11 32 47](https://user-images.githubusercontent.com/30245296/47421310-99262600-d780-11e8-877a-f99fb49293dc.png)
```
QGroupBox::title{
  ...
  background-color:qlineargradient(x1:0,y1:1,x2:0,y2:0,stop:0 rgba(100,100,100,255),stop:1 rgba(125,125,125,255));
  border:1px solid rgba(25,25,25,175);
  ...
}
```
A bit more elegant...

**Comment by bouncyball-git** [comment]
How can I change triangle color myself?

**Comment by masc4ii** [comment]
Change the two pngs in Photoshop or another program. (See one of the latest commits, where I changed it to white. Located in darktheme folder.)

**Comment by bouncyball-git** [comment]
Thanx

**Comment by bouncyball-git** [comment]
Tried lots of colors for triangle and I guess you are right, none of this improved visibility much.

Then I came up with this. Is it possible?
![edit_pane](https://user-images.githubusercontent.com/17232233/47440786-eecdf300-d7bf-11e8-894f-32682a88a1ac.png)

**Comment by masc4ii** [comment]
Hm... black is very black... should be 25,25,25, then it would fit better I think. I got it nearly, but I can't change the background. If I change it manually in the dock, the whole dock is black. If I change it via qss, nothing happens. If I change it via MainWindow, all is black, but the dock is unchanged. GRRRRR. The rest works:
![bildschirmfoto 2018-10-24 um 20 02 13](https://user-images.githubusercontent.com/30245296/47451420-fe513a00-d7c7-11e8-9335-109c7c50473f.png)

**Comment by masc4ii** [comment]
Now I got it. But position slider looks not good now...
![bildschirmfoto 2018-10-24 um 20 19 52](https://user-images.githubusercontent.com/30245296/47452319-3f4a4e00-d7ca-11e8-88d6-36f3f1d26d04.png)

**Comment by bouncyball-git** [comment]
It look very cool and simple! Bad for info panel though :(

So what can be done further?

**Comment by masc4ii** [comment]
It just affects the dialog background, the rest is identical. But this one had to be darker for the groupboxes... is as it is...

I would create a new menu action, where you can switch between these two designs, if it is okay for you (sry, but I still like the old more, even if this one looks really not bad). Don't know if that works, but it should be possible. It will be easier than the old one too, because I think I don't need two qss for Win/OSX (overlapping don't cares here).

**Comment by bouncyball-git** [comment]
Yes! gonna be great. But if brings in a lot of trouble just forget about it :)

**Comment by masc4ii** [comment]
Added it for you. Now we can switch around... yeah.

**Comment by bouncyball-git** [comment]
Hey man thank you for doing it for me very much!!! How to use this feature?

**Comment by masc4ii** [comment]
You're welcome. Under View you'll find Theme. There you can toggle between the two themes.

**Comment by bouncyball-git** [comment]
Hehe my bad :). Thanks again it now looks more like material design! Love it!

**Comment by bouncyball-git** [comment]
How many pixels (gap) are between group tabs now?

**Comment by bouncyball-git** [comment]
And one more thing it really asks to make line brakes some light gray instead of white as they are now.

**Comment by masc4ii** [comment]
You mean the separator lines? Should be possible somehow. 
Space is defined in ui by layout group.

**Comment by bouncyball-git** [comment]
Hmm... strange. Will try tomorrow.

**Comment by masc4ii** [comment]
![bildschirmfoto 2018-10-29 um 10 19 28](https://user-images.githubusercontent.com/30245296/47640627-41166780-db64-11e8-862b-6570f4031066.png)
![bildschirmfoto 2018-10-29 um 10 19 49](https://user-images.githubusercontent.com/30245296/47640631-42e02b00-db64-11e8-8869-589f827a201a.png)
Manually changed in ui. That seems to be the only way. Do you like it? Then I commit it...

**Comment by bouncyball-git** [comment]
Oh YES! :)

**Comment by bouncyball-git** [comment]
Hey man talking about design. I have a small idea regarding fu..ing/bloody  dual iso section ;)

Finally I want to account the DISO block presence. So I think there can be these changes in the edit pane:

1. When DISO block is not detected all widgets of dual iso part will be grayed out, excep the small check box with title "Force dual iso" or "Old dual iso" or similar which fits in the interface better, you decide, you got the idea (this is for older dual iso MLVs without DISO block or ones which are somewhat stubborn to detect)
2. When DISO block is detected all widgets of dual iso block will be active except "Force" check box which will be grayed out.

What do you think about it?

**Comment by bouncyball-git** [comment]
Btw can we make separator lines a bit darker? With small diff to tab back color.

**Comment by masc4ii** [comment]
Sure. Do you like to play and tell me which color you like? Now we have 3x128.

**Comment by bouncyball-git** [comment]
May be this.

![newlinecolor](https://user-images.githubusercontent.com/17232233/47706597-13542000-dc43-11e8-80f8-4f97cc4a3b5d.png)

**Comment by masc4ii** [comment]
What color is it?

**Comment by bouncyball-git** [comment]
IDK ;) just did this in gimp. Could this be picked by pipette? Thickness... yeah it is 3pix now right?

**Comment by masc4ii** [comment]
Sure... done.
